src/utils/downloadCsv.ts

```diff
@@ -0,0 +1,53 @@
+import type { CardItem } from '@/api/types';
+import { formatDate } from '@/utils/formatDate';
+
+function formatCsvCell(cell: string | number | null): string {
+  const str = String(cell);
+  if (str.includes(',') || str.includes(' ') || str.includes('\n')) {
+    return `"${str.replace(/"/g, '""')}"`;
+  }
+  return str;
+}
+
+export const downloadCsv = (items: CardItem[]) => {
```

Comment:
You've already manually added the " quotes to the headers, and then you're also running them through formatCsvCell, which also adds quotes if needed - this can lead to extra double quotes in the CSV